api.js

```diff
@@ -46,10 +47,40 @@ function postCommentByArticleId(article_id,username,commentBody){
     
   })
 }
+
+function deleteCommentByCommentId(comment_id){
+   return axios
+  .delete(`https://be-backend-project-nc-news.onrender.com/api/comments/${comment_id}`).then(({data})=>{
+      return data
+  })
+}
+
+function fetchTopics(){
+
+  return axios
+  .get(`https://be-backend-project-nc-news.onrender.com/api/topics`).then(({data:{topics}})=>{
+     return topics
+  })
+
+}
+
+
+function fetchAllArticlesByTopic(sortByQuery) {
+
+  return axios
+    .get(`https://be-backend-project-nc-news.onrender.com/api/articles?topic=${sortByQuery}`)
```

Comment:
You should be able to do this via your get all articles request with axios params. If you keep them separate, it's going to come up against obstacles later when you add your sort_bys